pkg/scheduler/framework/types.go

```diff
@@ -215,6 +215,8 @@ type WeightedAffinityTerm struct {
 type Diagnosis struct {
 	NodeToStatusMap      NodeToStatusMap
 	UnschedulablePlugins sets.String
+	// PreFilterMsg records the messages returned from PreFilterPlugins.
```

Comment:
```suggestion
	// PreFilterMsg records the messages returned from PreFilter plugins.
```

(Could you also update PostFilterPlugin to PostFilter plugins?)

Reply:
Made the suggested changes.